docs/root/version_history/current.rst

```diff
@@ -134,6 +135,8 @@ Deprecated
 * http: the HeaderMatcher fields :ref:`exact_match <envoy_v3_api_field_config.route.v3.HeaderMatcher.exact_match>`, :ref:`safe_regex_match <envoy_v3_api_field_config.route.v3.HeaderMatcher.safe_regex_match>`,
   :ref:`prefix_match <envoy_v3_api_field_config.route.v3.HeaderMatcher.prefix_match>`, :ref:`suffix_match <envoy_v3_api_field_config.route.v3.HeaderMatcher.suffix_match>` and
   :ref:`contains_match <envoy_v3_api_field_config.route.v3.HeaderMatcher.contains_match>` are deprecated by :ref:`string_match <envoy_v3_api_field_config.route.v3.HeaderMatcher.string_match>`.
+* http: :ref:`append <envoy_v3_api_field_config.core.v3.HeaderValueOption.append>` field has been deprecated in favor of the new
+  :ref:`append_action <envoy_v3_api_field_config.core.v3.HeaderValueOption.append_action>` field which is more clear and can be used to further extend the header matcher functionality.
```

Comment:
nit: can you see what functionality you actually added and ref link to that specifically?